rust/cargo-psibase/src/package.rs

```diff
@@ -205,7 +205,7 @@ pub async fn build_package(
     metadata: &MetadataIndex<'_>,
     service: Option<&str>,
 ) -> Result<PackageInfo, anyhow::Error> {
-    let mut depends = vec![];
+    let mut depends_set = HashSet::new();
```

Comment:
The elements are already being sorted. Adding a `dedup` at that point is probably more efficient.